config/initializers/devise.rb

```diff
@@ -142,7 +142,7 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 8..128
+  config.password_length = (Utils.if_present(ENV['MIN_PASSWORD_LENGTH'], :to_i) || 8)..128
```

Comment:
I added the `if_present` helper to avoid that case. When the argument is not `present?` it returns `nil`. It does the same as `ENV['MIN_PASSWORD_LENGTH'].present? ? ENV['MIN_PASSWORD_LENGTH'].to_i : 8`, but I did not want to repeat that pattern a couple of times.